### DISCUSSION TITLE
Docker hub instance

### DISCUSSION THREAD

**Discussion by spudnik267** [external-secrets/kubernetes-external-secrets#929] [General]
is externalsecrets/kubernetes-external-secrets on docker hub the same as this repo?

**Comment by Flydiverny** [comment]
Yes, it does however not have all versions, but I mirrored the available versions at some point.

**Reply by Flydiverny** [reply]
See this for additional alternatives:
- https://github.com/external-secrets/kubernetes-external-secrets/issues/928
